src/components/Categories/MainCategoryTile.vue

```diff
@@ -1,85 +1,73 @@
 <template>
-	<div v-if="tileSize === 'large'" class="tw-mb-6">
-		<kv-responsive-image class="category-image"
-			:images="retinaImageExists ?
-				[['small', image],
-					['small retina', retinaImage]]
-				:
-				[['small', image]]"
-			loading="lazy"
-			:alt="imageAlt"
-		/>
-		<large-tile-info
-			:category-name="categoryName"
-			:category-description="categoryDescription"
-			:number-loans="numberLoans"
-		/>
-	</div>
-	<div v-else-if="tileSize === 'medium'" class="tw-mb-6">
-		<kv-responsive-image class="category-image"
-			:images="retinaImageExists ?
-				[['small', image],
-					['small retina', retinaImage]]
-				:
-				[['small', image]]"
-			loading="lazy"
-			:alt="imageAlt"
-		/>
-		<div class="tw-hidden md:tw-block">
-			<h3 class="tw-mt-2 tw-mb-2">
-				Lend to {{ categoryName }}
-			</h3>
+	<div v-if="tileSize === 'large' || tileSize === 'medium'" class="tw-mb-6">
+		<a
```

Comment:
Please use the `router-link` component instead of and `a` link.

Reply:
I originally used router-link when I was linking to /lend-by-category/${categoryName}, but when we switched to using the full link from graphQL with this line below:
return this.categoryUrl.replace('lend', 'lend-by-category');

it appended the entire url like this:
https://dev-vm-01.kiva.org/https://dev-vm-01.kiva.org/lend-by-category/women

Reply:
So I could only get these two options to work.

<router-link
:to="`/lend-by-category/${categoryName}`"

or 

<a
:href="cleanURL"

Reply:
The `categoryName` does not always match the end of the url. 

Lets use `<router-link>`  with `:to=/lend-by-category/${last part of the URL in the API response string}`

So you'll have to modify the `cleanURL` computed variable to get that last segment of the url.

Reply:
Yeah I initially was replacing all spaces and periods with a dash in categoryName:

<router-link
:to="/lend-by-category/${cleanCategoryTile}"

cleanCategoryTile() {
const cleanedTitle = this.categoryName.replace(/[\s.]/g, '-').toLowerCase();
if(cleanedTitle.slice(-1) === '-') {
return cleanedTitle.slice(0, -1);
}
return cleanedTitle;
}


I'll try taking out the last part of the URL from the API instead tomorrow. Thank you!